pandas/core/series.py

```diff
@@ -2492,9 +2493,9 @@ def map(self, arg, na_action=None):
 
         See Also
         --------
-        Series.apply: For applying more complex functions on a Series
-        DataFrame.apply: Apply a function row-/column-wise
-        DataFrame.applymap: Apply a function elementwise on a whole DataFrame
+        Series.apply : For applying more complex functions on a Series.
+        DataFrame.apply : Apply a function row-/column-wise.
+        DataFrame.applymap : Apply a function elementwise on a whole DataFrame.
```

Comment:
Can you also add Series.replace ?

Reply:
This is outside the original scope of the contribution, see first PR comment.